v2.0/admin-ui-node-map.md

```diff
@@ -0,0 +1,185 @@
+
+- Monitor disk utilization to know if you need to add more storage capacity.
+- Identify dead nodes in the cluster.
+- Be informed about unavailable or under-replicated ranges.
+- Confirm that configuration changes were persisted to the cluster.
+- Observe QPS across nodes to ensure that queries are rerouted to live nodes if a node failure occurs.
+
+## Configure and Navigate the Node Map
+
+To configure the **Node Map**, you need to start the cluster with the correct `--localities` flags and assign the latitudes and longitudes for each region.
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> won't be displayed until all regions are assigned the corresponding latitudes and longitudes. {{site.data.alerts.end}}
+
+Consider a four-node geo-distributed cluster with the following configuration:
+
+|  Node | Region | Datacenter |
+|  ------ | ------ | ------ |
+|  Node1 | us-east | us-east-1 |
+|  Node2 | us-east | us-east-1 |
+|  Node3 | us-west | us-west-1 |
+|  Node4 | eu-west | eu-west-1 |
+
+#### Step 1. Ensure the CockroachDB version is 2.0 or higher
+
+~~~ shell
+$ ./cockroach version
+~~~
+
+If not, [upgrade to CockroachDB v2.0](upgrade-cockroach-version.html).
+
+#### Step 2. Start the nodes with the correct `--localities` flags
+
+In a new terminal, start Node 1:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach start \
+--insecure \
+--locality=region=us-east,datacenter=us-east-1  \
+--store=node1 \
+--host=localhost \
+--port=26257 \
+--http-port=8080 \
+--join=localhost:26257,localhost:26258,localhost:26259
+~~~
+
+In a new terminal, start Node 2:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach start \
+--insecure \
+--locality=region=us-east,datacenter=us-east-1 \
+--store=node2 \
+--host=localhost \
+--port=26258 \
+--http-port=8081 \
+--join=localhost:26257,localhost:26258,localhost:26259
+~~~
+
+In a new terminal, start Node 3:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach start \
+--insecure \
+--locality=region=us-west,datacenter=us-west-1 \
+--store=node3 \
+--host=localhost \
+--port=26259 \
+--http-port=8082 \
+--join=localhost:26257,localhost:26258,localhost:26259
+~~~
+
+In a new terminal, start Node 4:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach start \
+--insecure \
+--locality=region=eu-west,datacenter=eu-west-1 \
+--store=node4 \
+--host=localhost \
+--port=26260 \
+--http-port=8083 \
+--join=localhost:26257,localhost:26258,localhost:26259
+~~~
+
+In a new terminal, use the [`cockroach init`](initialize-a-cluster.html) command to perform a one-time initialization of the cluster:
+
+{% include copy-clipboard.html %}
+~~~ shell
+./cockroach init --insecure
+~~~
+
+#### Step 3. [Set the enterprise license](enterprise-licensing.html)
+
+#### Step 4. Insert the latitudes and longitudes of each region into the `systems.locations` table
+
+{% include copy-clipboard.html %}
+~~~ sql
```

Comment:
it's obvious to me that this snippet should be copied into a sql shell, but is that obvious to everyone?